study-builder/studies/basil/file-uploads.conf

```diff
@@ -197,6 +201,8 @@
             "stableId": "FILE_UPLOAD_3",
             "isRestricted": false,
             "isDeprecated": false,
+            "maxFileSize": 2097152,
+            "mimeTypes": [],
```

Comment:
Does it make sense to add at least one fileUpload with 2-3 mime-types defined?
For example:
`"mimeTypes": [ "text/plain", "text/html" ],`
Of course it can be added later by someone else..

Reply:
That will be nice. Point of this PR is to just get things in place. Next PRs can do more.